src/views/Dashboard/GeneticAnalyst/Layout.vue

```diff
@@ -215,7 +215,7 @@ export default {
 
     goToDashboard() {
       this.showModalError = false
-      this.$router.push({ name: "dashboard"})
+      this.$router.push({ name: "dashboard", query: {}})
```

Comment:
Why is this pushing an empty query? @azmimuhammad

Reply:
on this PR https://github.com/debionetwork/debio-genetic-analyst-ui/pull/13 push query if status response 4/5, if not pushing an empty query the error popup will always active